Github Actions, Sync Workflow fails with "Error: Process completed with exit code 12."

**Describe the bug**
Sync Workflow is failing on archive files with zip error.

**To Reproduce**
Steps to reproduce the behavior:
1. Manually run Sync Workflow in Github Actions.

**Expected behavior**
Workouts to be uploaded to Garmin

**Logs**
From "archive files":
...
Preparing to unpack .../unzip_6.0-26+deb11u1_amd64.deb ...
Unpacking unzip (6.0-26+deb11u1) ...
Selecting previously unselected package zip.
Preparing to unpack .../archives/zip_3.0-12_amd64.deb ...
Unpacking zip (3.0-12) ...
Setting up unzip (6.0-26+deb11u1) ...
Setting up zip (3.0-12) ...

**zip error: Nothing to do! (try: zip -r -j output.zip . -i /app/output)
Error: Process completed with exit code 12.**

**Installation (please complete the following information):**
 - Method: [Github Actions]
 - Version: v3.5.1

--- a/.github/workflows/sync_peloton_to_garmin.yml
+++ b/.github/workflows/sync_peloton_to_garmin.yml
@@ -90,14 +90,14 @@ jobs:
         TZ: America/Chicago
 
     - name: archive files
-      if: ${{ github.event.inputs.saveLocalCopy }}
+      if: ${{ github.event.inputs.saveLocalCopy == 'true' }}
       run: |
         apt-get update
         apt-get install -y zip
         zip -r -j output.zip ${{ env.OUTPUT_DIR }}
         
     - uses: actions/upload-artifact@v3
-      if: ${{ github.event.inputs.saveLocalCopy }}
+      if: ${{ github.event.inputs.saveLocalCopy == 'true'}}
       with:
         name: output
         path: output.zip

--- a/.github/workflows/sync_peloton_to_garmin.yml
+++ b/.github/workflows/sync_peloton_to_garmin.yml
@@ -90,15 +90,15 @@ jobs:
         TZ: America/Chicago
 
     - name: archive files
-      # saveLocalCopy can be set when workflow action is manually run otherwise it's skipped 
-      if: ${{ github.event.inputs.saveLocalCopy == 'true' || false }}
+      # saveLocalCopy can be set when workflow action is manually run otherwise it's always skipped 
+      if: ${{ github.event.inputs.saveLocalCopy == 'true' }}
       run: |
         apt-get update
         apt-get install -y zip
         zip -r -j output.zip ${{ env.OUTPUT_DIR }}
         
     - uses: actions/upload-artifact@v3
-      if: ${{ github.event.inputs.saveLocalCopy == 'true' || false }}
+      if: ${{ github.event.inputs.saveLocalCopy == 'true' }}
       with:
         name: output
         path: output.zip

--- a/.github/workflows/sync_peloton_to_garmin.yml
+++ b/.github/workflows/sync_peloton_to_garmin.yml
@@ -90,14 +90,15 @@ jobs:
         TZ: America/Chicago
 
     - name: archive files
-      if: ${{ github.event.inputs.saveLocalCopy == 'true' }}
+      # saveLocalCopy can be set when workflow action is manually run otherwise it's skipped 
+      if: ${{ github.event.inputs.saveLocalCopy == 'true' || false }}
       run: |
         apt-get update
         apt-get install -y zip
         zip -r -j output.zip ${{ env.OUTPUT_DIR }}
         
     - uses: actions/upload-artifact@v3
-      if: ${{ github.event.inputs.saveLocalCopy == 'true'}}
+      if: ${{ github.event.inputs.saveLocalCopy == 'true' || false }}
       with:
         name: output
         path: output.zip

